src/module_type_translator.ts

```diff
@@ -97,6 +97,9 @@ export class ModuleTypeTranslator {
    */
   symbolsToAliasedNames = new Map<ts.Symbol, string>();
 
+  /** A cache for expensive symbol lookups, see TypeTranslator.symbolToString. */
```

Comment:
Can you make this comment say something about what the variable is?  E.g. the values in the map are the closure translated names for the symbols.

Reply:
Done.